### ISSUE TITLE
Objects can be selected only in hierarchy

### ISSUE THREAD

**Issue by romanokeser** [SebLague/Ray-Marching#7]
Hi, a dummy here. Why can't I select the objects in the Scene? They can be only selected in the hierarchy window ¯\_(ツ)_/¯

**Comment by IgnacyGadzina** [comment]
You can't select, because unity doesn't support raymarching. The shader is selfmade by SebLague. All you can is just see it.

**Comment by romanokeser** [comment]
> You can't select, because unity doesn't support raymarching. The shader is selfmade by SebLague. All you can is just see it.

Can you ELI5, please? How's it that Unity is doesn't support raymarching, there are tons of links and documentation on the web
